### ISSUE TITLE
Inputting an external mermaid file throws a syntax error

### ISSUE THREAD

**Issue by harrelfe** [quarto-dev/quarto-cli#3665]
### Bug description

Consider this external file named `mermaid1.mer`:

```
flowchart LR
  A
```

When running `Quarto` 1.3.55 on the following script I get `Syntax error in graph, mermaid version 9.1.1`:

````
---
title: "Test"
format:
  html:
    self-contained: true
---

```{mermaid}
%%| file: mermaid1.mer
```
````

I'm running Ubuntu 22.

### Checklist

- [X] Please include a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [X] Please [format your issue](https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us) so it is easier for us to read the bug report.
- [X] Please document the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [X] Please document the operating system you're running. If on Linux, please provide the specific distribution.

**Comment by niksirbi** [comment]
I have the exact same issue as @harrelfe on `ubuntu 22.10`. I am using `Quarto v1.2.280` in `VSCode v1.74.2` to make a `revealjs` presentation. 

Loading a file thus
````{markdown}
```{mermaid}
%%| file:  diagram.mmd
```
````
results in a `Syntax error in graph, mermaid version 9.1.1`.

Puzzlingly, copy-pasting the contents of that `diagram.mmd` in the mermaid code block works as intended.
````{markdown}
```{mermaid}
flowchart LR
    A --> B
```
````

I also tried the following:
````{quarto}
```{mermaid}
{% include diagram.mmd %}
```
````
but I end up with the exact same error.

**Comment by ProtossGP32** [comment]
I'm facing the same issue. Trying to embed a simple `test.mmd` file that contains:
```
classDiagram
    class test{
    }
```
into a `.qmd` file throws a `Syntax error in graph mermaid version 9.2.0` error:

```
---
title: "Page title"
categories: [category1, category2]
---

```{mermaid}
%%| label: fig-uml
%%| fig-cap: "UML diagram"
%%| file: test.mmd
```
```
NOTE: Sorry about the last line, I don't know why it is incorrectly rendering the last code snippet

**Comment by mrstlee** [comment]
Had the same problem with 1.2.280 on my macbook pro.
Upgrading to 1.3.340 with https://github.com/quarto-dev/quarto-cli/releases/download/v1.3.340/quarto-1.3.340-macos.pkg fixed it.

**Comment by niksirbi** [comment]
Thanks @mrstlee, upgrading to quarto 1.3.340 on my Macbook Pro solved the issue for me as well!

**Comment by valcu** [comment]
This issue is still here with Ubuntu 22.04, VSC: 1.80.2, and Quarto: 1.2.335

**Comment by mcanouil** [comment]
@valcu as stated in the issue? This is fixed in 1.3. You have to upgrade to get the fix.

**Comment by mcanouil** [comment]
Quarto VSCode extension is not the same thing as Quarto CLI.
